doc/website/getting-started/installation.md

```diff
@@ -160,11 +155,13 @@ If you want to execute only individual testcases then you can use these executab
 
 ## Use Sanitizer Scan
```

Comment:
I'd like to keep the getting started section as simple and easy as possible. A sanitizer scan is definitely something advanced.

@dkroenke Can we move this section to an `advanced-installation.md` or possibly `CONTRIBUTING.md`?

Reply:
`advanced-installation.md` makes more sense

Reply:
Is that something you want to have on [iceoryx.io](http://iceoryx.io)? IMHO nope, but we could add a link.

Reply:
> Is that something you want to have on [iceoryx.io](http://iceoryx.io)? IMHO nope, but we could add a link.

why not? Let's keep the users as long on our site as possible.

Reply:
Move the section to `website/advanced/installation-for-developers.md`